.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -32,22 +33,22 @@ body:
     attributes:
       label: Pitch
       description: A clear and concise description of what you want to happen.
-      placeholder: ""
+      placeholder:
```

Comment:
should we just remove the placeholders that are now empty?

Reply:
sorted out here [9061750](https://github.com/SAFEHR-data/PIXL/pull/514/commits/90617507131d1366017b075312dac717425a6e68)